packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -7772,6 +7772,48 @@ QUnit.module('label integration', () => {
 });
 
 QUnit.module('accessibility', () => {
+    QUnit.test('Tags aria labels should match inputAttr aria-label mappings', function(assert) {
+        const productsData = [{
+            ID: 1,
+            Name: '1',
+        }, {
+            ID: 2,
+            Name: '2',
+        }];
+        const $tagBox = $('#tagBox').dxTagBox({
```

Comment:
```suggestion
        const tagBox = $('#tagBox').dxTagBox({
```
Because your version will return instance of TagBox

Reply:
[9ccdab0](https://github.com/DevExpress/DevExtreme/pull/27157/commits/9ccdab096e8605104508353289768770d18bdf2a)